crates/bevy_pbr/src/render/pbr_lighting.wgsl

```diff
@@ -244,7 +244,7 @@ fn spot_light(
     R: vec3<f32>, F0: vec3<f32>, diffuseColor: vec3<f32>
 ) -> vec3<f32> {
     // reuse the point light calculations
-    let point = point_light(world_position, light, roughness, NdotV, N, V, R, F0, diffuseColor);
+    let point_light = point_light(world_position, light, roughness, NdotV, N, V, R, F0, diffuseColor);
```

Comment:
Is there any problem with having variables named the same as functions? I guess not.

Reply:
works for me at least... and point is now a reserved keyword
```
ailed to create shader module: Validation Error

Caused by:
    In Device::create_shader_module

Shader '' parsing error: name `point` is a reserved keyword
    ┌─ wgsl:486:9
    │
486 │     let point = point_light(world_position, light, roughness, NdotV, N, V, R, F0, diffuseColor);
    │         ^^^^^ definition of `point`


    name `point` is a reserved keyword
```